tests/test_web/test_api_v1.py

```diff
@@ -504,6 +505,7 @@ def test_add_bundle_success(
             'url': 'http://localhost/api/v1/builds/1/logs',
             'expiration': '2020-02-15T17:03:00Z',
         },
+        'omps_operator_version': {},
```

Comment:
May I ask why is the org not None but still the `omps_operator_version` missing? Could you please add the test where both of them are not None?

Reply:
I guess here is tested just the first response (and not waiting for finals state). `omps_operator_version` is filled after OMPS is API called and I guess this is not the test for it.

The test for this is covered in our integration test (iib-api-tests).

Reply:
You can probably mock the response of the OMPS call and return a mock value, no?